sarsen/apps.py

```diff
@@ -67,6 +68,52 @@ def product_info(
     return product_info
 
 
+def gamma_on_overlapping_blocks(
+    data: xr.DataArray,
+    correct_radiometry: str,
+    chunks: int = 500,
+    bound: int = 30,
+    **kwargs,
+) -> xr.DataArray:
+
+    if correct_radiometry == "gamma_bilinear":
+        gamma_weights = radiometry.gamma_weights_bilinear
+    elif correct_radiometry == "gamma_nearest":
+        gamma_weights = radiometry.gamma_weights_nearest
+
+    blocks_x = []
+    for nx in np.arange((data.x.size - bound) // chunks + 1):
```

Comment:
@aurghs to align to the exterior of the blocks to the data boundaries, I would have expected `2 * bound`. No?